client/blocks/image-editor/test/image-editor-canvas.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+import { useSandbox } from 'test/helpers/use-sinon';
+
+class ImageEditorCropMock extends Component {
+	constructor( props ) {
+		super( props );
+	}
+	render() {
+		return <dfn />;
+	}
+}
+
+describe( 'ImageEditorToolbar', () => {
+	useFakeDom();
```

Comment:
Ah I suppose the editor makes some assumptions about the window object being available?